packages/core/admin/admin/src/content-manager/components/AttributeFilter/index.js

```diff
@@ -1,13 +1,81 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
+import { useQueryParams, getDisplayName } from '@strapi/helper-plugin';
+
+import { useAdminUsers } from '../../../hooks/useAdminUsers';
 import useAllowedAttributes from './hooks/useAllowedAttributes';
 import Filters from './Filters';
+import { AdminUsersFilter } from './AdminUsersFilter';
+
+const customOperators = [
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$eq', defaultMessage: 'is' },
+    value: '$eq',
+  },
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$ne', defaultMessage: 'is not' },
+    value: '$ne',
+  },
+];
+
+const AUTHOR_ATTRIBUTES = ['createdBy', 'updatedBy'];
+
+const getUsersSelected = (query) => {
+  return (
+    query?.filters?.$and?.reduce((acc, filter) => {
+      const [key, value] = Object.entries(filter)[0];
+      const id = value.id?.$eq || value.id?.$ne;
+
+      if (AUTHOR_ATTRIBUTES.includes(key) && !acc.includes(id)) {
+        acc.push(id);
+      }
+
+      return acc;
+    }, []) ?? []
+  );
+};
+
+const formatUsers = (users, formatMessage) => {
+  return users.map((user) => ({
+    label: getDisplayName(user, formatMessage),
+    customValue: user.id.toString(),
+  }));
+};
```

Comment:
IMHO since you only use this once I would inline both these functions – clearer where they're used and how they're used :)